framer/Components/RangedSliderComponent.coffee

```diff
@@ -0,0 +1,337 @@
+Utils = require "../Utils"
+{Layer} = require "../Layer"
+{Events} = require "../Events"
+
+"""
+RangedSliderComponent
+
+minKnob <layer>
+maxKnob <layer>
+knobSize <width, height>
+fill <layer>
+
+min <number>
+max <number>
+minValue <number>
+maxValue <number>
+
+pointForValue(<n>)
+valueForPoint(<n>)
+
+animateToMinValue(value, animationOptions={})
+animateToMaxValue(value, animationOptions={})
+"""
+
+Events.SliderValueChange  = "sliderValueChange"
+Events.SliderMinValueChange = "sliderMinValueChange"
+Events.SliderMaxValueChange = "sliderMaxValueChange"
+
+class Knob extends Layer
```

Comment:
Yo dawg. I see you like classes. But really, you should capture reusability like this in a simple function. Less code, less overhead.

```
makeKnob = -> new Layer({...})

```